.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] The commit message follows our guidelines: https://github.com/single-spa/single-spa-angular/blob/master/CONTRIBUTING.md#commit
```

Comment:
Isn't this unnecessary since the pre-commit hook and build will fail if they don't do this?

Reply:
Good point